Generals/Code/Tools/CMakeLists.txt

```diff
@@ -1,22 +1,29 @@
 # Build tools
 
-add_subdirectory(Autorun)
-add_subdirectory(Babylon)
-add_subdirectory(buildVersionUpdate)
-add_subdirectory(Compress)
-add_subdirectory(CRCDiff)
-add_subdirectory(DebugWindow)
-add_subdirectory(GUIEdit)
-add_subdirectory(ImagePacker)
-add_subdirectory(Launcher)
-add_subdirectory(mangler)
-add_subdirectory(MapCacheBuilder)
-add_subdirectory(matchbot)
-add_subdirectory(ParticleEditor)
-add_subdirectory(PATCHGET)
-add_subdirectory(textureCompress)
-add_subdirectory(timingTest)
-add_subdirectory(versionUpdate)
-add_subdirectory(wolSetup)
-add_subdirectory(WorldBuilder)
-add_subdirectory(WW3D)
+# Build useful stuff.
+if(GENZH_BUILD_GENERALS_TOOLS)
+    add_subdirectory(DebugWindow)
+    add_subdirectory(GUIEdit)
+    add_subdirectory(ImagePacker)
+    add_subdirectory(MapCacheBuilder)
+    add_subdirectory(ParticleEditor)
+    add_subdirectory(WorldBuilder)
+endif()
+
+# Build tat.
```

Comment:
tat?

Better: "Build less useful tools"

Reply:
Its colloquial yorkshire for "useless things".